tools/src/test/java/org/neo4j/tools/org/neo4j/index/GBPTreePlayground.java

```diff
@@ -0,0 +1,233 @@
+        @Override
+        public void run( String[] args, PrintStream out ) throws Exception
+        {
+            Long[] longValues = new Long[args.length];
+            for ( int i = 0; i < args.length; i++ )
+            {
+                longValues[i] = Long.valueOf( args[i] );
+            }
+            MutableLong key = new MutableLong();
+            MutableLong value = new MutableLong();
+            try ( Writer<MutableLong,MutableLong> writer = tree.writer() )
+            {
+                for ( Long longValue : longValues )
+                {
+                    key.setValue( longValue );
+                    value.setValue( longValue );
+                    writer.put( key, value );
+                }
+            }
+            catch ( IOException e )
+            {
+                throw new RuntimeException( e );
+            }
+            maybePrint();
+        }
+
+        @Override
+        public String toString()
+        {
+            return "N [N ...] (add key N)";
+        }
+    }
+
+    private class RemoveCommand implements Command
+    {
+        @Override
+        public void run( String[] args, PrintStream out ) throws Exception
+        {
+            Long[] longValues = new Long[args.length];
+            for ( int i = 0; i < args.length; i++ )
+            {
+                longValues[i] = Long.valueOf( args[i] );
+            }
+            MutableLong key = new MutableLong();
+            try ( Writer<MutableLong,MutableLong> writer = tree.writer() )
+            {
+                for ( Long longValue : longValues )
+                {
+                    key.setValue( longValue );
+                    writer.remove( key );
+                }
+            }
+            catch ( IOException e )
+            {
+                throw new RuntimeException( e );
+            }
+            maybePrint();
+        }
+
+        @Override
+        public String toString()
+        {
+            return "N [N ...] (remove key N)";
+        }
+    }
+
+    private class ToggleAutoPrintCommand implements Command
+    {
+        @Override
+        public void run( String[] args, PrintStream out )
+        {
+            if ( autoPrint.isTrue() )
+            {
+                autoPrint.setFalse();
+            }
+            else
+            {
+                autoPrint.setTrue();
+            }
+        }
+
+        @Override
+        public String toString()
+        {
+            return "Toggle auto print after modifications (ON by default)";
+        }
+    }
+
+    private void maybePrint() throws IOException
+    {
+        if ( autoPrint.getValue() )
+        {
+            print();
+        }
+    }
+
+    private void print() throws IOException
+    {
+        tree.printTree();
+    }
+
+    public static void main( String[] args ) throws InterruptedException, IOException
+    {
+        FileSystemAbstraction fs = new DefaultFileSystemAbstraction();
+        File indexFile = new File( "index" );
```

Comment:
I would like it if it would first look at `args` to see if its length > 0 and if so grab the first arg as the file (could be absolute). It was the first thing I ran into when trying this out. It could also perhaps print which file (absolute path) it's working on when starting up.

And should it really delete the file in the end, always? What if want to try out what happens between restarts, with generations and all that?